readthedocs/api/v2/serializers.py

```diff
@@ -90,6 +90,9 @@ class Meta:
             'built',
             'downloads',
             'type',
+            'has_pdf',
+            'has_epub',
+            'has_htmlzip',
```

Comment:
shouldn't we add these to fields to APIv3 Version detail endpoint as well?

I'm not sure, since the download link will appear only if it exists. So, maybe it's redundant.

Reply:
These `has_*` fields feel like redundant fields for our API

Reply:
I agree it is a little awkward. These fields are used for writing by the build servers but not reading by the general public.